src/index.js

```diff
@@ -4,12 +4,15 @@ const { JSDOM, ResourceLoader } = require('jsdom');
 // Accepts a jsdom config object.
 module.exports = class Window {
 	constructor(jsdomConfig = {}) {
-		const { proxy, strictSSL, userAgent } = jsdomConfig;
-		const resources = new ResourceLoader({
-			proxy,
-			strictSSL,
-			userAgent
-		});
+		let resources = jsdomConfig.resources;
```

Comment:
You can use object destructuring here:

```suggestion
		let {resources} = jsdomConfig;
```